services/ske/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v0.11.0 (2024-03-28)
+
+- Waiters for async operation `StartCredentialsRotationWaitHandler` and `CompleteCredentialsRotationWaitHandler`
```

Comment:
```suggestion
- **Feature**: Waiters for async operation `StartCredentialsRotationWaitHandler` and `CompleteCredentialsRotationWaitHandler`
```

Reply:
Also not sure we need the link to the example, as not all new features will be included in examples